docs/_user_guide/best_practices_gui.md

```diff
@@ -1,237 +1,73 @@
+
+* For **Windows**, we recommend installing [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install) for a full-featured Linux experience on Windows.
+      
+    ??? "WSL Quick Installation Instructions"
+          * run `wsl --install`
+          * After WSL installs restart computer
+          * Open search and look for Ubuntu
+          * Follow the instructions to set up your username and password
+          * You can now use the Ubuntu shell directly
 
-- For **Windows**, you need to have [XMing](https://sourceforge.net/projects/xming/) installed first.
 
 #### Login to Yens with X-forwarding
-```
+```title="Terminal Command"
 ssh -Y <SUNet ID>@yen.stanford.edu
 ```
 
 The `-Y` flag here enables X-forwarding, allowing you to run software on the server that uses graphical windows.
 
 When prompted, type your SUNet ID password. Then, complete the two-factor authentication process.
-After you successfully login, check that X-forwarding works correctly. Choose any of the following commands and type it in
-the yen command line interface as all of them will pop up a window if everything is working correctly - `xeyes`, `xcalc`, `xlogo` or `xclock`
-(or choose your favorite X11 command line program).
+After you successfully login, check that X-forwarding works correctly. Choose any of the following commands and type it in the yen command line interface as all of them will pop up a window if everything is working correctly - `xeyes`, `xcalc`, `xlogo` or `xclock` (or choose your favorite X11 command line program).
 
-```bash
+```title="Terminal Command"
 xeyes
 ```
 which will pop up a window with eyes tracking where your mouse is.
 
-![](/images/xeyes.png)
+![](/assets/images/xeyes.png)
 
 ### Examples
-#### R / RStudio GUI
-
-To run RStudio GUI, load the R module with the version of R that you want:
-
-```bash
-ml R/4.0.0
-```
-
-List the loaded modules:
-
-```bash
-module list
-
-Currently Loaded Modules:
-  1) rstudio/1.1.463   2) R/4.0.0
-```
-
-We see that RStudio is loaded automatically when you load R module. We can now launch RStudio with:
-
-```bash
-rstudio
-```
-
-The RStudio GUI will pop open and you can do anything you would normally do in RStudio (but with noticeable delay due to X tunneling):
-
-![](/images/rstudio-gui.png)
-
-
-If you forget to load R module or loaded only the rstudio module, you will see an error that R is not found:
-
-![](/images/rstudio-err.png)
-
-If that happens, just load `R` module and start RStudio again.
-#### MATLAB
-To run MATLAB GUI, load the matlab module with the version that you want:
-
-```bash
-ml matlab
-```
-
-We can now launch MATLAB GUI with:
-
-```bash
-matlab
-```
-
-The MATLAB GUI will pop open and you can do anything you would normally do (but with noticeable delay due to X tunneling):
-
+!!! tip "Notebooks can be used as an Alterative to a traditional GUI"
```

Comment:
weird capitalization in the tip...either make the first word capital or all :)
Alternative is also misspelled.